### DISCUSSION TITLE
CLI

### DISCUSSION THREAD

**Discussion by project-sbc** [nefarius/HidHide#30] [Q&A] [answered]
Saw the CLI option in the readme, wasn't sure how to access it. I installed via msi installer. Any advice would be appreciated. 

Looking to implement on handheld gaming devices like GPD Win 3, Aya neo, would like an easy way to quickly enable and disable the built in controller. Would like to be able to use CLI

Thanks and regards

Edit: just saw the issues tab with the CLI comment, this can be considered answered

**Comment by nefarius** [comment, marked as answer]
Yep, not yet released 😊
